src/main.ts

```diff
@@ -1,16 +1,12 @@
 import { BunRuntime } from "@effect/platform-bun";
 import { Layer } from "effect";
-import { MessagePubSub } from "./pubsub/message-pubsub";
 import { SpotifyService } from "./spotify/spotify-service";
-import { TwitchService } from "./twitch/twitch-service";
+import { TwitchServiceLive } from "./twitch/twitch-service";
 
 const BunTime = {
   funTime: BunRuntime.runMain,
 };
 
-const MainLive = Layer.provide(
-  Layer.mergeAll(TwitchService, SpotifyService),
-  MessagePubSub.Live,
```

Comment:
We provided this to the services locally, so it can removed from the main layer.